src/components/admin/DepartmentContact.vue

```diff
@@ -1,25 +1,28 @@
 <template>
-  <v-expansion-panel :id="`department-contact-${contact.id}`" class="panel-width">
+  <v-expansion-panel
+    :id="`department-contact-${contact.id}`"
+    :aria-labelledby="`department-contact-${contact.id}-btn`"
+    class="panel-width"
+    role="region"
+  >
     <v-expansion-panel-title :id="`department-contact-${contact.id}-btn`" class="pa-2 rounded-b-0 height-unset">
       <span v-if="isEditing" class="sr-only">Edit</span>
       <div :id="`dept-contact-${contact.id}-name`" class="font-weight-bold">{{ fullName }}</div>
+      <span class="sr-only">department contact</span>
     </v-expansion-panel-title>
-    <v-expansion-panel-text
-      :aria-labelledby="`department-contact-${contact.id}-btn`"
-      class="edit-contact-container"
-      role="region"
-    >
+    <v-expansion-panel-text class="edit-contact-container">
       <v-container
         v-if="!isEditing"
         :id="`dept-contact-${contact.id}-details`"
         class="pb-0 px-0"
         fluid
       >
         <v-row :id="`dept-contact-${contact.id}-email`">
-          <v-col class="px-2 pt-1" cols="12">{{ contact.email }}</v-col>
+          <v-col class="px-2 pt-1" cols="12"><span class="sr-only">Email address:</span>{{ contact.email }}</v-col>
```

Comment:
@lyttam - Do we want a space after the colon, so it reads clearly?

Reply:
@johncrossman I think Vue inserts one automatically. I was testing using JAWS and it sounded fine.